packages/orchestration/src/orchestration-response.ts

```diff
@@ -51,6 +55,54 @@ export class OrchestrationResponse {
     return choice?.message?.content;
   }
 
+  /**
+   * Messages that can be used for subsequent prompts as message history.
+   * @param choiceIndex - The index of the choice to parse.
+   * @returns A list of all messages.
+   */
+  getAllMessages(choiceIndex = 0): ChatMessages {
```

Comment:
Just checked the other PR https://github.com/SAP/ai-sdk-js/pull/538/files#r1955748987

Don't forget to adjust readme.